### PR TITLE
dev-lang/rust-bin: keywords mask -999 & -9999 ebuilds

### PR DESCRIPTION
**Pull request by okias**
-999* should be keyword unmasked only when they're needed.

### CODE REVIEW

**Comment by stefson** [comment]
have you tested if this still works without `RESTRICT="network-sandbox"`?

**Comment by okias** [comment]
I haven't :-(
just repoman was bashing about it.

if ot's needed, I'll put it back

**Comment by stefson** [comment]
have about _you_ actually test it?

**Comment by cnd** [comment]
@okias I'm not sure if that will work without `RESTRICT="network-sandbox"` it's something that needs to be tested.

**Comment by okias** [comment]
rebased with network sandbox restrict kept

**Comment by stefson** [comment]
Why do you even want to drop the keywords?

File: dev-lang/rust/rust-999.ebuild [hunk 1]
```diff
@@ -14,6 +14,7 @@ EGIT_BRANCH="beta"
 
 EGIT_CHECKOUT_DIR="${MY_P}-src"
 KEYWORDS=""
+RESTRICT="network-sandbox"
```
Comment (gentoo90) [inline dev-lang/rust/rust-999.ebuild:17]: No, wrong file. `rust` does not need this. `EGIT` source code downloading works without `network-sandbox`. This belongs to `rust-bin`.

**Comment by okias** [comment]
@stefson because live ebuilds (-999, -9999, -99999999) by Gentoo policy shouldn't have KEYWORDS

**Comment by stefson** [comment]
yeah, but these are bin packages, they don't compile anything.

**Comment by okias** [comment]
as I understand, it's not versioned. So when I install once -999 and keep it, it'll not offer me update, when content of package changes. So users who once install -999 will not get updates, unless he explicitly asks for it. That's an issue. If you unmasked live package you have knowledge what you doing, but this without masking can get installed without much notice.

**Pull request closed without merging**